v22.2/alter-index.md

```diff
@@ -19,6 +19,10 @@ Subcommand | Description
 [`SPLIT AT`](split-at.html) | Force a [range split](architecture/distribution-layer.html#range-splits) at the specified row in the index.
 [`UNSPLIT AT`](unsplit-at.html) | Remove a range split enforcement in the index.
 
+## Index visibility
+
+Use `[NOT] VISIBLE` to set whether an index is visible to the [cost-based optimizer](cost-based-optimizer.html#control-whether-the-optimizer-uses-an-index). If not visible, the index will not be used in queries unless specifically selected with [index hint](indexes.html#selection). For an example, see [Set an index to be not visible](#set-an-index-to-be-not-visible).
```

Comment:
I wanted to double-check whether `[NOT] VISIBLE` is a standard way of representing syntax like this, as opposed to "`VISIBLE` and `NOT VISIBLE`". I haven't seen the former used elsewhere in our docs, but may have missed a similar usage.

For example:
```suggestion
Use `VISIBLE` or `NOT VISIBLE` to set whether an index is visible to the [cost-based optimizer](cost-based-optimizer.html#control-whether-the-optimizer-uses-an-index). If `NOT VISIBLE`, the index will not be used in queries unless specifically selected with [index hint](indexes.html#selection). For an example, see [Set an index to be not visible](#set-an-index-to-be-not-visible).
```